src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -3715,7 +3774,11 @@ void MacroAssembler::kernel_crc32(Register crc, Register buf, Register len,
   Label L_by16, L_by16_loop, L_by4, L_by4_loop, L_by1, L_by1_loop, L_exit;
 
   if (UseCRC32) {
-      kernel_crc32_using_crc32(crc, buf, len, table0, table1, table2, table3);
+      if (UseCryptoPmullForCRC32) {
```

Comment:
Whis is this inside `UseCRC32`? It means that if we set `-XX:-UseCRC32 -XX:+UseCryptoPmullForCRC32` on the command line, we won't get `UseCryptoPmullForCRC32`

Reply:
crc32 instructions are used to process remaining input not multiple of 128 bytes. The dependency between these two options is removed.